src/components/case_manager/types/table.ts

```diff
@@ -0,0 +1,45 @@
+export interface Person {
```

Comment:
It would be useful to have these interfaces in a more central location. 
@codetheweb: any ideas on where to put them?

Reply:
Fixed everything except this one. We can either wait for a response from Max or change it later. What do you think?

Reply:
all the types in this file are already present in `src/backend/db/zapatos/schema.d.ts`

we can remap them slightly if necessary or re-export them to make things more ergonomic, but we need to stick with a single source of truth for database types

for example:
```ts
import { SelectableForTable } from "zapatos/schema"
type User = SelectableForTable<"user">;
```

Reply:
Thanks for pointing that out. Didn't realize those existed already. I'll likely import them into this file, so that the HeadCell interface can still access them, and re-export for the Table.